src/LinkDotNet.Blog.Web/Features/Admin/BlogPostEditor/Components/CreateNewBlogPost.razor

```diff
@@ -1,108 +1,120 @@
-			<div class="alert alert-info text-muted form-text mt-3 mb-3">
-				The first page of the blog is cached. Therefore, the blog post is not immediately visible.
-				Head over to <a href="/settings">settings</a> to invalidate the cache or enable the checkmark.
-				<br>
-				The option should be enabled if you want to publish the blog post immediately and it should be visible on the first page.
-			</div>
-			<div class="form-check form-switch mb-3">
-				<InputCheckbox class="form-check-input" id="invalidate-cache" @bind-Value="model.ShouldInvalidateCache"/>
-				<label class="form-check-label" for="invalidate-cache">Make it visible immediately</label><br/>
-			</div>
-		</div>
-	</EditForm>
+    <h3 class="fw-bold">@Title</h3>
+    <EditForm Model="@model" OnValidSubmit="OnValidBlogPostCreatedAsync">
+        <DataAnnotationsValidator />
+        <div class="form-floating mb-3">
+            <input type="text" class="form-control" id="title" placeholder="Title"
+                   @oninput="args => model.Title = args.Value!.ToString()!" value="@model.Title" />
+            <label for="title">Title</label>
+            <ValidationMessage For="() => model.Title"></ValidationMessage>
+        </div>
+        <div class="form-floating mb-3">
+            <MarkdownTextArea Id="short" Class="form-control" Rows="4" Placeholder="Short Description"
+                              @bind-Value="@model.ShortDescription"
+                              PreviewFunction="ReplaceShortCodes"></MarkdownTextArea>
+            <ValidationMessage For="() => model.ShortDescription"></ValidationMessage>
+        </div>
+        <div class="form-floating mb-3 relative">
+            <MarkdownTextArea Id="content" Class="form-control" Rows="20" Placeholder="Content"
+                              PreviewFunction="ReplaceShortCodes"
+                              @bind-Value="@model.Content"></MarkdownTextArea>
+            <ValidationMessage For="() => model.Content"></ValidationMessage>
+
+            <div class="btn-group position-absolute bottom-0 end-0 m-5 extra-buttons">
+                <button class="btn btn-primary btn-outlined btn-sm dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false">
+                    More
+                </button>
+                <ul class="dropdown-menu">
+                    @if (shortCodes.Count > 0)
+                    {
+                        <li>
+                            <button type="button" @onclick="OpenShortCodeDialog" class="dropdown-item">
+                                <span>Get shortcode</span>
+                            </button>
+                        </li>
+                    }
+                    <li><button type="button" class="dropdown-item" @onclick="FeatureDialog.Open">Experimental Features</button></li>
+                </ul>
+            </div>
+            <div class="btn-group position-absolute bottom-0 m-5 extra-buttons">
```

Comment:
I would like to put the whole button inside the "More" Dropdown. Otherwise there are too many buttons as "overlay".

Reply:
Much better - I like